### DISCUSSION TITLE
Suggests for GRDB and LNASTATE values for SDRPLay sdr?

### DISCUSSION THREAD

**Discussion by jeff-luszcz** [jvde-github/AIS-catcher#126] [General]
The SDRPlay sdr has two configuration settings GRDB and LNASTATE that allow tuning of the devices reception.

There's not a lot of documentation about what good values for these settings should be in their documentation.

Any suggestions for good values for these in AIS Catcher?

I'm currently using the default values and see my signal level between -50 and -60 db

**Comment by jvde-github** [comment]
I saw this [tweet](https://twitter.com/snebur_is_right/status/1652326734883176452) and with these settings I had 3-4 times the number of messages than with the default settings (don't think the -a switch is implemented for the SDRplay). 

At the end of the day I think it is experimenting what works best for your system. But these might work actually quite well.

Perhaps should consider to make these the default....

**Comment by jeff-luszcz** [comment]
Thanks, unfortunately Twitter is no longer showing tweets if you don't have an account (and nitter.net doesn't show that tweet either) Could you past the info into the thread here and I'll give them a try and report back? Thanks!

**Comment by jvde-github** [comment]
```
AIS-catcher -gs GRDB 45 LNASTATE 3 AGC off
```
Let me know if this works for you as well?

**Comment by jeff-luszcz** [comment]
Thanks! Just swapped over and will let it run. It's a holiday week here so less commercial ship traffic in the harbor but I expect to see better Signal Levels perhaps....

**Comment by jeff-luszcz** [comment]
hmmmm, I've let it run for a little while and am not getting the results I hoped for.

I set the parameters to 

-gs GRDB 45 LNASTATE 3 AGC off 

while also using -s 3072K

As you can see the results dropped 10 days ago when I changed to the LNASTATE value and then it popped back up when I removed the value and went back to to the defaults.

Not sure what to make of this (and it reflects my experience playing around with the values before asking the question too, weird.

![Screen Shot 2023-07-12 at 12 51 13 PM](https://github.com/jvde-github/AIS-catcher/assets/8672214/50788f1b-4bf7-4162-be07-37c5fe100741)

**Comment by jvde-github** [comment]
Ok, that is a pity. At the end of the day it is very station and setup specific and requires systematic experimentation.

For me the settings worked well, but having said that, when I removed the AIS filter from the chain, the settings were not working so well anymore which makes sense as a totally different gain is likely needed.
